tests/FINDOLOGIC/Export/Tests/ItemTest.php

```diff
@@ -34,23 +35,27 @@ private function getMinimalItem()
         return $item;
     }
 
-    /**
-     * @expectedException \FINDOLOGIC\Export\Data\EmptyElementsNotAllowedException
-     */
     public function testAddingEmptyAttributesCauseException()
     {
-        $item = $this->getMinimalItem();
-        $attribute = new Attribute('empty attribute', []);
-        $item->addAttribute($attribute);
+        try {
+            $item = $this->getMinimalItem();
+            $attribute = new Attribute('empty attribute', []);
+            $item->addAttribute($attribute);
+        } catch (EmptyElementsNotAllowedException $e) {
+            $expectedMessage = "Elements with empty values are not allowed. 'Attribute' with the name 'empty attribute'";
+            $this->assertEquals($expectedMessage, $e->getMessage());
+        }
     }
 
-    /**
-     * @expectedException \FINDOLOGIC\Export\Data\EmptyElementsNotAllowedException
-     */
     public function testAddingEmptyPropertiesCauseException()
     {
-        $item = $this->getMinimalItem();
-        $property = new Property('empty property', []);
-        $item->addProperty($property);
+        try {
+            $item = $this->getMinimalItem();
+            $property = new Property('empty property', []);
+            $item->addProperty($property);
```

Comment:
Test does not fail if no exception is thrown. Needs a `$this->fail(...)` here.

Reply:
done